apps/accelerate/speedster/speedster/api/tests/test_pytorch.py

```diff
@@ -111,14 +116,49 @@ def test_torch_torchscript():
     res_original = model(x)
     res_optimized = optimized_model(x)[0]
 
-    assert isinstance(optimized_model, PytorchBackendInferenceLearner)
+    assert isinstance(optimized_model, TorchScriptInferenceLearner)
+    assert torch.max(abs((res_original - res_optimized))) < 1e-2
+
+
+@pytest.mark.skipif(
+    not check_module_version(torch, min_version="2.0.0") or True,
```

Comment:
Aren't you always skipping this test?

Reply:
Yep, it's being skipped because torch dynamo is a work in progress